GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -322,6 +312,16 @@ public FormBrowse(GitUICommands commands, string filter, ObjectId? selectedId =
             RevisionGrid.FirstId = firstId;
 
             InitializeComplete();
+
+            ToolStripFilters.Bind(() => Module, RevisionGrid);
+            ToolStripFilters.SetRevisionFilter(filter);
+
+            _aheadBehindDataProvider = GitVersion.Current.SupportAheadBehindData ? new AheadBehindDataProvider(() => Module.GitExecutable) : null;
+
+            repoObjectsTree.Initialize(_aheadBehindDataProvider, branchFilterAction: ToolStripFilters.SetBranchFilter, RevisionGrid, RevisionGrid, RevisionGrid);
+            revisionDiff.Bind(RevisionGrid, fileTree, () => RequestRefresh());
+            fileTree.Bind(() => RequestRefresh());
+
```

Comment:
Move these because `InitializeComplete` invokes translation engine, which applies translations to UI elopements, and thus can corrupt state, e.g. a toolbar button may be set to "itemB", but the translation engine will re-apply "itemA".

TODO: verify it works correctly in non-English scenarios.

Reply:
I am unable to verify non English loc's due to https://github.com/axuno/SmartFormat/issues/192. So this is something we'll have to verify at a later stage.

Reply:
I've tested Russian with https://github.com/gitextensions/gitextensions/pull/9542, and everything works as expected.